purchases/src/main/kotlin/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -98,6 +98,16 @@ class StoreTransaction(
      */
     val subscriptionOptionId: String?,
 
+    /**
+     * The id of the SubscriptionOption purchased for each product ID.
+     *
+     * In Google, this will be calculated from the basePlanId and offerId
+     * Null in Google for restored transactions and purchases initiated outside of the app.
+     * Null for Amazon purchases.
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    val subscriptionOptionIdForProductIDs: Map<String, String>?,
```

Comment:
[Not related to this line!] 

Just thinking, for offline entitlements mode, we might need to also change the code to consider all the purchased subscriptions? TBH, I'm not familiarized in how these purchases are queried from Google, so if they appear as separate purchases, it might be ok as is, but wanted to confirm.